src/dsp_tools/models/value.py

```diff
@@ -26,11 +26,15 @@ def __str__(self) -> str:
     def get_all_iris(self) -> Optional[list[str]]:
         return self.__iriregexp.findall(self.__xmlstr)
 
-    def replace(self, fromStr: str, toStr: str) -> None:
-        self.__xmlstr = self.__xmlstr.replace(fromStr, toStr)
-
-    def regex_replace(self, pattern: str, repl: str) -> None:
-        self.__xmlstr = regex.sub(pattern=repr(pattern)[1:-1], repl=repl, string=self.__xmlstr)
+    def find_all_iri_in_xmlstr(self) -> set[str]:
```

Comment:
This method name is misleading: It doesn't return IRIs, but IDs. I suggest something like: `find_ids_referenced_in_salsah_links()`